src/libraries/System.Private.CoreLib/src/System/SearchValues/SearchValuesDebugView.cs

```diff
@@ -13,6 +15,7 @@ public SearchValuesDebugView(SearchValues<T> values)
             _values = values;
         }
 
+        [DebuggerBrowsable(DebuggerBrowsableState.RootHidden)]
```

Comment:
Does the IDE revert back to showing the root if there are a lot of values? Otherwise, it may be noisy.

Reply:
It creates a scrollable list.